Configure.cmake

```diff
@@ -78,49 +98,46 @@ CHECK_C_SOURCE_COMPILES("
 
 set (CMAKE_REQUIRED_FLAGS ${FLAGS_ENABLE_FMA4})
 CHECK_C_SOURCE_COMPILES("
-#if defined(_MSC_VER)
-#include <intrin.h>
-#else
-#include <x86intrin.h>
-#endif
-int main() {
-  __m256d r = _mm256_macc_pd(_mm256_set1_pd(1), _mm256_set1_pd(2), _mm256_set1_pd(3));
-}" COMPILER_SUPPORTS_FMA4)
+  #if defined(_MSC_VER)
+  #include <intrin.h>
```

Comment:
Sorry, I have kind of lost track of these changes and it would take significant time for me to revert these with big impact on changes I have made that are based on this patch :(.
I am aware of my mistake and I apologise for the unrelated changes. Could we please let these in this patch and I will make sure no more of this happens. Thanks.